guide/src/logging.md

```diff
@@ -0,0 +1,60 @@
+# Logging
+
+It is desirable if both the Python and Rust parts of the application end up
+logging using the same configuration into the same place.
+
+This can trivially be accomplished by not logging at all in one of them. If
```

Comment:
I think this sentence doesnt add so much - users presumably come here because they want to do something rather than not log at all :smile: 

I'm tempted to suggest this paragraph gets replaced with a new one that describes what the rest of this chapter will be, e.g.

```suggestion
This section of the guide briefly discusses how to connect the two languages' logging ecosystems together. The recommended way for Python extension modules is to configure Rust's logger to send log messages to Python using the `pyo3-log` crate. For users who want to do the opposite and send Python log messages to Rust, see the note at the end of this guide.
```

This will help readers find what section of this chapter could be relevant to them by just scanning the introduction.
 
Of course please feel free to change what's written here it you don't like this suggestion!